.travis.yml

```diff
@@ -14,16 +14,20 @@ env:
     - TESTING=ON TESTOPTS="-E \(imc\|spce_cma_simple\)" CMAKE_BUILD_TYPE=Debug
     - TESTING=OFF CMAKE_BUILD_TYPE=Debug   WERROR=yes
     - TESTING=ON TESTOPTS="-E \(imc\|spce_cma_simple\)" CMAKE_BUILD_TYPE=Release DOCKERHUB=yes
+    - DISTRO=fedora_gmx5.1 TESTING=ON TESTOPTS="-E \(imc\|spce_cma_simple\)" CMAKE_BUILD_TYPE=Release
```

Comment:
Yes, but all tests get executed all the time! (I am sick of maintaining 5 versions of `.travis.yml`!)